tradeexecutor/state/visualisation.py

```diff
@@ -53,6 +53,11 @@ class PlotShape(enum.Enum):
     #:
     #: See https://plotly.com/python/line-charts/?_ga=2.83222870.1162358725.1672302619-1029023258.1667666588#interpolation-with-line-plots
     horizontal_vertical = "hv"
+    
+    #: Individually specified points.
+    #: 
+    #: Typically used for event indicators e.g. cross over of two lines.
+    marker = "markers"
```

Comment:
Be consistant that enum name and value are the same unless there is a reason them to be different.

Otherwise users might have unexpected surprises e.g. when doing search and replace, diagnostics